### PR TITLE
kubeadm: added tests cert/pkiutil pkg

### PR DESCRIPTION
**Pull request by dmmcquay**
raised coverage from ~37% to ~77%

**What this PR does / why we need it**: added tests to cert/pkiutil pkg and raised coverage from ~37% to ~77%

Adding unit tests is a WIP from #34136

**Special notes for your reviewer**: /cc @luxas @pires 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by dmmcquay** [comment]
@mikedanese mind reviewing this PR?

File: cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers_test.go [hunk 1]
```diff
@@ -107,3 +107,101 @@ func TestWriteCertAndKey(t *testing.T) {
 		)
 	}
 }
+
+func TestCertOrKeyExist(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir")
+	}
+	defer os.Remove(tmpdir)
```
Comment (pires) [inline cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers_test.go]: Use `os.RemoveAll`.

File: cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers_test.go [hunk 2]
```diff
@@ -107,3 +107,101 @@ func TestWriteCertAndKey(t *testing.T) {
...
+		}
+	}
+}
+
+func TestTryLoadCertAndKeyFromDisk(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir")
+	}
+	defer os.Remove(tmpdir)
```
Comment (pires) [inline cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers_test.go]: Use `os.RemoveAll`.

**Review by pires — CHANGES_REQUESTED** [review]

**Comment by luxas** [comment]
I let @pires once he's happy with this

/approve

**Comment by dmmcquay** [comment]
@pires can I get your lgtm for this?

**Comment by pires** [comment]
@dmmcquay please squash.

**Comment by pires** [comment]
@k8s-bot cvm gce e2e test this

**Pull request merged**